### ISSUE TITLE
Resolving the current working directory against the static-base-uri isn't always correct

### ISSUE THREAD

**Issue by ndw** [docbook/xslTNG#241]
In previous releases, the `ext:cwd()` function has returned a path. The stylesheets resolve that against the static base URI to make an absolute URI. But if the static base URI is, for example, `https://cdn.docbook.org/...`, then you get a very strange answer for "current working directory."

The function already returns a URI, not a string. I think the solution is to have it return `file:/path/` instead of `/path/`. And `file:/c:/path` on Windows because that's my best guess.

In other words, have it return an absolute file: URI for the current working directory. That seems like the better answer.
